src/main/java/org/traccar/directions/timeDistance/JsonTimeDistanceObjectMapper.java

```diff
@@ -0,0 +1,16 @@
+package org.traccar.directions.timeDistance;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public final class JsonTimeDistanceObjectMapper {
+
+    private JsonTimeDistanceObjectMapper() { };
+
+    private static ObjectMapper objectMapper = new ObjectMapper()
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
I think it's better to use same mapper for everything. Also, isn't there a way to specify it for the class instead of globally?